EIPS/eip-4610.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: 4610
+title: ERC-721 Delegator Extension
+description: Enables ERC-721 to be loaned
+author: AFKDAO (@AFKDAO)
+discussions-to: https://github.com/ethereum/EIPs/issues/4608
```

Comment:
Kindly move the discussion to Ethereum Magicians, as mentioned by Micah, in ref. to the eip-1
Then update the `discussions-to` pointing to the Ethereum Magicians' discussion link.
https://github.com/ethereum/EIPs/issues/4608#issuecomment-1001846604